src/components/UserList/index.tsx

```diff
@@ -175,6 +256,20 @@ const UserList: React.FC = () => {
         <Table.TBody>
           {data?.map((user) => (
             <tr key={`user-list-${user.id}`}>
+              <Table.TD>
+                {user.id !== currentUser?.id && (
```

Comment:
What do you think about not allowing modification of user id 1 here either? (And in the logic to select all users)